src/handbook/peopleops/expenses.md

```diff
@@ -4,32 +4,22 @@ navTitle: Expenses
 
 ## Expenses
 
-The company will cover any reasonable work-related expense, including ensuring you
-have the equipment (laptop/monitor/accessories) you need to do the job. As an
-all-remote company, we meet via Zoom. As such, we will cover the cost of a good
-webcam/microphone/lighting arrangement.
+FlowForge will reimburse you for any work-related expenses that are considered reasonable. This includes providing you with the necessary equipment, such as a laptop, monitor, and accessories, to effectively carry out your job responsibilities. As an all-remote company, our meetings are conducted using either Zoom or Slack. Therefore, we will also cover the expenses associated with obtaining a high-quality webcam, microphone, and lighting setup.
```

Comment:
```suggestion
FlowForge will reimburse you for any work-related expenses that are considered reasonable. This includes providing you with the necessary equipment, such as a laptop, monitor, and accessories, to effectively carry out your job responsibilities. As an all-remote company, our meetings are conducted using either Google Meet or Slack. Therefore, we will also cover the expenses associated with obtaining a high-quality webcam, microphone, and lighting setup.
```

Reply:
Reading this again, it feels like most of these lines should be moved to "Work & location" as section